### ISSUE TITLE
Campaign voiceover videos should be unzipped

### ISSUE THREAD

**Issue by expipiplus1** [FAForever/downlords-faf-client#1493]
**Describe the bug**

The in-scenario voiceover videos do not load. As the videos are not loaded the sounds for them play all at once when they are supposed to be played sequentially.

This thread on the forum describes the symptoms and a possible fix: https://forums.faforever.com/viewtopic.php?f=78&t=16674

These files should be unzipped once they have been downloaded.

**To Reproduce**

- Run one of the vanilla campaign missions 
- Observe that when a video is due to play the audio plays successfully but the video never appears
- Observe that sometimes several audio tracks play simultaneously 

**Expected behavior**

- Sequential videos play sequentially
- Audio does no overlay

**OS**
Linux

**Comment by expipiplus1** [comment]
See also https://github.com/FAForever/fa/issues/2788

**Comment by 1-alex98** [comment]
Fixed in 1.0.1RC2 only that the RC is broken, but that is a different issue

**Comment by expipiplus1** [comment]
Thanks!

**Comment by 1-alex98** [comment]
Go test that in RC4 , this should work now

**Comment by 1-alex98** [comment]
And tell me if it works ;)

**Comment by speed2CZ** [comment]
It does unpack them, but it unpacks them every time you host a mission, which is bad.

**Comment by 1-alex98** [comment]
ok yeah, lets fix that before i release it

**Comment by 1-alex98** [comment]
oh yeah i can not fix it

**Comment by 1-alex98** [comment]
I mean I need to unpack it, because I can not be sure if something changed right

**Comment by 1-alex98** [comment]
🤔 I mean I could assume that if the file is already on the disk that the (mpvies folder was unzipped, that I could do

**Comment by speed2CZ** [comment]
unpack after downloading it?

**Comment by 1-alex98** [comment]
yes will do that

**Comment by expipiplus1** [comment]
Thanks!
